build.gradle

```diff
@@ -60,7 +60,7 @@ compileTestGroovy {
 }
 
 def libertyAntVersion = "1.9.13"
-def libertyCommonVersion = "1.8.29"
+def libertyCommonVersion = "1.8.30-SNAPSHOT"
```

Comment:
Shouldn't you be pulling in a new ci.ant version too? You made a change there for Java 21.

Reply:
same comment for LMP btw